Source: written by Steffen Hau <steffen@hauihau.de>, Gentoo Bug #282573
Upstream: yes, caca Trac #61
Reason: fix compilation with freeglut-2.6
diff -Naur libcaca-0.99.beta16.orig/configure.ac libcaca-0.99.beta16/configure.ac
--- libcaca-0.99.beta16.orig/configure.ac
+++ libcaca-0.99.beta16/configure.ac
@@ -223,7 +223,7 @@
       break])
   if test "${ac_cv_my_have_gl}" = "yes"; then
     AC_DEFINE(USE_GL, 1, Define to 1 to activate the OpenGL backend driver)
-    GL_LIBS="${GL_LIBS} -lGL -lglut"
+    GL_LIBS="${GL_LIBS} -lGL -lGLU -lglut"
     CACA_DRIVERS="${CACA_DRIVERS} gl"
   elif test "${enable_gl}" = "yes"; then
     AC_MSG_ERROR([cannot find OpenGL+FreeGLUT development files])
